### DISCUSSION TITLE
Tesla Authentication Broken again ?

### DISCUSSION THREAD

**Discussion by hemang249** [timdorr/tesla-api#373] [Q&A] [answered]
Since this morning, my scripts for Tesla authentication have broken. I am getting timeout after timeout. Has the API broked down again? I have tried manipulating the headers but nothing has worked so far. Even the Tesla App was not loading for me, is it somthing on Tesla's end that has gone wrong?

**Comment by hemang249** [comment, marked as answer]
Ok I think it was an issue on Tesla's side, the Auth API is working today.

**Comment by just4give** [comment]
I am facing timeout error, first time running the script ( https://github.com/timdorr/tesla-api/discussions/378 ) ... is this the same error you were seeing other day ?

**Reply by hemang249** [reply]
Are you getting any error message from the API or it just hangs up ?

Edit: I am unsure regarding the ruby wrapper. I was using the  REST api itself to authenticate my users. All my scripts are currently functional. The issue i was facing was indeed on Tesla's end and not anything in the script I had.

**Reply by just4give** [reply]
Would you mind sharing your code if possible ?
